### PR TITLE
Move navigation to config

### PR DESCRIPTION
**Pull request by RobLoach**
This uses `src/config.yaml`, but we could possibly use `src/settings.yaml`? I found `src/globals.yaml` sounded more programmer-y rather than user-friendly-y. Thoguhts?

#47

### CODE REVIEW

**Comment by andrewmallis** [comment]
How about a navigation.yaml that contains our primary and footer navs
and a config.yaml to contain variables like the contact form email address, copyright year, etc?

**Comment by andrewmallis** [comment]
ref #43

**Comment by RobLoach** [comment]
Added it in. Looks pretty good now. Anything else you can think of to stick in there? We could add it in later on too.

File: src/navigation.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+top:
```
Comment (andrewmallis) [inline src/navigation.yaml]: top > primary (achieves more parity with Drupal and is more semantic)

**Pull request merged**